src/main.js

```diff
@@ -340,4 +354,28 @@ function init() {
     renderOverlay();
 }
 
+function getMaxPage() {
+    let pageCalc = Math.ceil(nonFollowersList.length / UNFOLLOWERS_PER_PAGE);
```

Comment:
Better to always use `const` instead of `let` unless the variable gets re-assigned :)